src/reliability.jl

```diff
@@ -0,0 +1,75 @@
+struct CronbachAlpha{T <: Real}
+    alpha::T
+    dropped::Vector{T}
+end
+
+function Base.show(io::IO, x::CronbachAlpha)
+    @printf(io, "Reliability for all items: %.4f", x.alpha)
```

Comment:
Maybe we'd better just say "Cronbach's alpha" rather than "reliability" here and below? Otherwise, nothing in the output indicates what's exactly the coefficient that we're reporting.

Reply:
Done!